src/bindings/js/node/include/core_wrap.hpp

```diff
@@ -68,7 +68,17 @@ class CoreWrap : public Napi::ObjectWrap<CoreWrap> {
     Napi::Value set_property(const Napi::CallbackInfo& info);
     Napi::Value get_property(const Napi::CallbackInfo& info);
 
+    /** @brief Query device if it supports the specified model with specified properties.
```

Comment:
Move this description to addon.ts
You will have to rewrite it according to [typedoc](https://typedoc.org/example/)
You can look at how other methods in addon.ts are described